gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/store/ManagedDirectBufferAllocator.java

```diff
@@ -60,6 +62,13 @@
   public static final String DIRECT_STORE_OBJECT_OWNER =
       "SNAPPYDATA_DIRECT_STORE_OBJECTS";
 
+  public static final String CACHED_DATA_FRAME_RESULTOUTPUT_OWNER =
+          "CACHED_DATA_FRAME_RESULTOUTPUT";
+
+  public static final List<String> nonEvictingOwners = new ArrayList() {{
+    add("CACHED_DATA_FRAME_RESULTOUTPUT_OWNER");
+  }};
```

Comment:
Yes. That was a bug i was struggling with. Thanks. This variable is used in CachedDataFrame while allocating off-heap memory by ByteBufferDataOutput.